stack.yaml

```diff
@@ -4,4 +4,5 @@ packages:
 - '.'
 extra-deps:
 - uri-0.1.6.4
+## INSERT CODECOV HERE -- DO NOT REMOTE THIS COMMENT. SEE ALSO ISSUE #156. ##
```

Comment:
`REMOTE` ... `REMOVE` ...?

Reply:
57741049aac7f9ad5dc34e5f1c0663d993a72686 fixed this.